src/ArmTemplates/Extractor/EntityExtractors/APIExtractor.cs

```diff
@@ -279,5 +280,32 @@ public async Task<ApiTemplateResources> GetApiRelatedTemplateResourcesAsync(stri
 
             return apiDependency;
         }
+
+        void ValidateOutputApiTemplateResource(string apiName, ApiTemplateResource apiResource, ApiTemplateResources apiTemplateResources)
```

Comment:
since it is clearing and modifying the api template resource, I would not just call it `Validate`. I can not come up with a better naming right now, but I would expect it to be called like `Sanitize` or I don't know

Reply:
Updated to suggested Sanitize :) Thank you!